fvtest/porttest/omrsockTest.cpp

```diff
@@ -254,36 +255,39 @@ TEST(PortSockTest, getaddrinfo_and_freeaddrinfo)
 	rc = OMRPORTLIB->sock_freeaddrinfo(OMRPORTLIB, resultPtr);
 	EXPECT_EQ(rc, OMRPORT_ERROR_INVALID_ARGUMENTS);
 
-	/* Get and verify that omrsock_getaddrinfo and omrsock_freeaddrinfo works. */
-	rc = OMRPORTLIB->sock_getaddrinfo(OMRPORTLIB, (char *)"localhost", NULL, hints, &result);
+	/* Verify that omrsock_getaddrinfo and omrsock_freeaddrinfo works with NULL hints. */
+	rc = OMRPORTLIB->sock_getaddrinfo(OMRPORTLIB, (char *)"localhost", NULL, NULL, &result);
 	ASSERT_EQ(rc, 0);
 
 	OMRPORTLIB->sock_getaddrinfo_length(OMRPORTLIB, &result, &length);
 	ASSERT_NE(length, 0);
 
-	for (uint32_t i = 0; i < length; i++) {
-		rc = OMRPORTLIB->sock_getaddrinfo_family(OMRPORTLIB, &result, &family, i);
-		EXPECT_EQ(rc, 0);
-		EXPECT_EQ(family, hintsFamily);
-
-		rc = OMRPORTLIB->sock_getaddrinfo_socktype(OMRPORTLIB, &result, &sockType, i);
-		EXPECT_EQ(rc, 0);
-		EXPECT_EQ(sockType, hintsSockType);
-
-		rc = OMRPORTLIB->sock_getaddrinfo_protocol(OMRPORTLIB, &result, &protocol, i);
-		EXPECT_EQ(rc, 0);
-	}
-	
 	OMRPORTLIB->sock_freeaddrinfo(OMRPORTLIB, &result);
 
-	/* Verify that omrsock_getaddrinfo and omrsock_freeaddrinfo works with NULL hints. */
-	rc = OMRPORTLIB->sock_getaddrinfo(OMRPORTLIB, (char *)"localhost", NULL, NULL, &result);
+	/* Get and verify that omrsock_getaddrinfo and omrsock_freeaddrinfo works. */
+	rc = OMRPORTLIB->sock_getaddrinfo(OMRPORTLIB, (char *)"localhost", NULL, hints, &result);
 	ASSERT_EQ(rc, 0);
 
 	OMRPORTLIB->sock_getaddrinfo_length(OMRPORTLIB, &result, &length);
 	ASSERT_NE(length, 0);
 
-	OMRPORTLIB->sock_freeaddrinfo(OMRPORTLIB, &result);
+	for (uint32_t i = 0; i < length; i++) {
+		EXPECT_EQ(OMRPORTLIB->sock_getaddrinfo_family(OMRPORTLIB, &result, &family, i), 0);
+		EXPECT_EQ(OMRPORTLIB->sock_getaddrinfo_socktype(OMRPORTLIB, &result, &sockType, i), 0);
+		EXPECT_EQ(OMRPORTLIB->sock_getaddrinfo_protocol(OMRPORTLIB, &result, &protocol, i), 0);
```

Comment:
```suggestion
		EXPECT_EQ(OMRPORTLIB->sock_addrinfo_family(OMRPORTLIB, &result, &family, i), 0);
		EXPECT_EQ(OMRPORTLIB->sock_addrinfo_socktype(OMRPORTLIB, &result, &sockType, i), 0);
		EXPECT_EQ(OMRPORTLIB->sock_addrinfo_protocol(OMRPORTLIB, &result, &protocol, i), 0);
```

Reply:
If it's okay, can we leave this fix for now? The Fix Addrinfo PR has code that fixes part of this. I will update it and fix it completely once the addrinfo fix is merged.